.changeset/many-moments-juggle.md

```diff
@@ -1,7 +1,12 @@
 ---
 "@fluidframework/container-loader": minor
 ---
+---
+section: deprecation
```

Comment:
Do we have a defined set of sections that we expect people to use? Maybe we should write them out (or at least, some common examples) in the changesets wiki.

Reply:
It's fully configurable. When creating changesets the new tool will prompt for the section. Options are dynamic based on the config.

Reply:
Yes, Tyler added them [as config in the root fluidBuild.config.js](https://github.com/microsoft/FluidFramework/blob/5504a9f6470d732861770c2ebb610687fc01875e/fluidBuild.config.cjs#L555-L563) recently. I agree adding a link to those in the wiki would be nice. But also note that if people generate changesets with `pnpm changeset` (vs manually copy pasting an existing one), the tool will give you the options based on that config. If I've been understanding the recent changes correctly 😄 .